### ISSUE TITLE
Possibility to mark current day

### ISSUE THREAD

**Issue by damiandudycz** [simla-tech/Fastis#28]
I would love to see the feature to customize the look of current day in the month view. I didn't find anything related in the configuration. Do you think you could add that feature?

**Comment by ilia3546** [comment]
Hello! You can customize view of the current day via `FastisConfig.DayCell` object. For example:

```swift
FastisConfig.default.dayCell.dateLabelColor = .black
FastisConfig.default.dayCell.selectedLabelColor = .green
```

You can find the full list of available properties [there](https://github.com/simla-tech/Fastis/blob/master/Sources/Views/DayCell.swift#L328)

**Comment by ilia3546** [comment]
Resolved in PR #36
